docs/dev/coding_standards.rst

```diff
@@ -0,0 +1,50 @@
+===============================
+🖍️ Coding Standards Reference Doc
+===============================
+
+*"Although that way may not be obvious at first unless you're Dutch."* - the Zen of Python
+
+
+Summary
+=======
+
+The goal is consistency. In style, we're in the right even if we're all wrong together. GeoIPS code is meant to be re-used by many, and we expect that the code will be read much more than it is written. The goal of this document is to help us spend less time on conventions and more time making GeoIPS better.
```

Comment:
Speaking of standards, do we want to enforce 88 characters for RST as well?

Speaking of which, do we have a good RST linter? I never found one I was happy with to use in CI, and it would probably be good to get something set up sooner than later...  The only linting of RST is validation by sphinx.

Reply:
There's also this: https://github.com/PyCQA/doc8 which seems more popular

Reply:
Just FYI, doc8 uses 79 characters as the maximum line length by default. We'd probably want to override to 88 characters to allow code blocks from our code. We might even want to allow a little wider to allow indented code blocks like this:

```rst
- A bullet
  - Another bullet describing a code block
    ```
    from geoips import interfaces
  
  
    interfaces.list()
    ```
```

Reply:
See https://github.com/NRLMMD-GEOIPS/geoips/issues/479 for the new issue